lib/util/UserInputNorm.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:decimal/decimal.dart';
+
+BigInt normalizeInput(double input) {
+  Decimal inDecimal = Decimal.parse(input.toString()) * Decimal.fromInt(10).pow(18);
```

Comment:
Can you comment on how this function works especially what fromInt and pow are doing